website/docs/r/compute_vm_instance.html.markdown

```diff
@@ -249,7 +266,10 @@ Review the argument references that you can specify for your resource.
 - `public_subnet` - (Optional, Forces new resource, String) The public subnet for the public network interface of the instance. Accepted values are primary public networks. You can find accepted values in the [subnets doc](https://cloud.ibm.com/classic/network/subnets).
 - `private_subnet` - (Optional, Forces new resource, String) The private subnet for the private network interface of the instance. Accepted values are primary private networks. You can find accepted values in the [subnets doc](https://cloud.ibm.com/classic/network/subnets).
 - `public_bandwidth_limited` - (Optional, Forces new resource, Integer) Allowed public network traffic in GB per month. It can be greater than 0 when the server is a monthly based server. Defaults to the smallest available capacity for the public bandwidth are used.  **Note** Conflicts with `private_network_only` and `public_bandwidth_unlimited`.
-- `public_bandwidth_unlimited` - (Optional, Forces new resource, Bool)   Allowed unlimited public network traffic in GB per month for a monthly based server. The `network_speed` should be 100 Mbps. Default value is **false**. **Note** Conflicts with `private_network_only` and `public_bandwidth_limited`.
+- `public_bandwidth_unlimited` - (Optional, Forces new resource, Bool) Allowed unlimited public network traffic in GB per month for a monthly based server. The `network_speed` should be 100 Mbps. Default value is **false**. **Note** Conflicts with `private_network_only` and `public_bandwidth_limited`.
+- `reserved_capacity_id` - (Optional, Forces new resource, Integer) Reserve capacity Id to provision this instance into.
```

Comment:
Reserve capacity Id to provision this instance into.
as
The reserve capacity ID to provision the instance.